tsconfig.test.json

```diff
@@ -6,6 +6,7 @@
     "jest.config.ts"
   ],
   "compilerOptions": {
+    "skipLibCheck": true,
```

Comment:
This is the error I get on Github checks if I don't put this line
https://github.com/ethersphere/gateway-proxy/actions/runs/3102646170/jobs/5025158837#step:11:9

Reply:
Interesting, that might be related to the upgraded TypeScript. I will have a look at it, but generally, it is preferable to solve the issue (for example some times an update of the dependency can help) rather than silent the checks for all the dependencies.

Reply:
So I have tried it myself and it works (see [here](https://github.com/ethersphere/gateway-proxy/pull/329/commits/0fcdda4d09d693387acc3b65c5163456e9559f0a)). Had to `npm install` as package-lock was inconsistent, so maybe there was the problem)...